default.nix

```diff
@@ -111,7 +111,7 @@ stdenv.mkDerivation rec {
   setupHook = writeText "setupHook.sh" "
     addCoqPath () {
       if test -d \"$1/lib/coq/${coq-version}/user-contrib\"; then
-        export COQPATH=\"$COQPATH\${COQPATH:+:}$1/lib/coq/${coq-version}/user-contrib/\"
+        export COQPATH=\"\${COQPATH-}\${COQPATH:+:}$1/lib/coq/${coq-version}/user-contrib/\"
```

Comment:
I am assuming that this reflects a similar change that happened in nixpkgs. Is that correct?

Reply:
Yes, I’ve just updated the PR description.